pom.xml

```diff
@@ -179,13 +179,17 @@
 
     <hamcrest.version>1.3</hamcrest.version>
     <lombok.version>1.12.6</lombok.version>
-    <resteasy.version>2.3.7.Final</resteasy.version>
     <findbugs.maven.version>3.0.1</findbugs.maven.version>
     <groovy.version>1.8.9</groovy.version>
     <guava.version>13.0.1</guava.version>
     <jackson.version>1.9.13</jackson.version>
-    <powermock.version>1.5.4</powermock.version>
+    <jaxb.api.version>2.2.12</jaxb.api.version>
+    <javadoc.version>2.9.1</javadoc.version>
+    <jxr.version>2.5</jxr.version>
+    <powermock.version>1.6.2</powermock.version>
+    <resteasy.version>3.0.13.Final</resteasy.version>
```

Comment:
I don't think resteasy is used in the client any more, so we should remove resteasy from zanata-parent and put it all in the server pom.